zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -915,6 +915,14 @@ property, when available, is noted below.
     ZooKeeper when loading database from disk, and syncing with leader.
     By default, this feautre is disabled, set "true" to enable it.
 
+* *outstandingHandshake.limit* 
```

Comment:
Sounds like something that should be turned on all the time with some meaningful default value. What do you think about setting it to 250 by default and let this property to override it.

Reply:
This feature hasn't been fully rollout internally, I would suggest to enable this by default, when we verified on our internal environment for a while, so if there is bug we can find it and fix it before its affecting the community.

Reply:
This is a master-only patch which doesn't even have an alpha-release, so I believe we could be a little bit more flexible about that, but I'm also happy with leaving it off.